drop-in/src/main/java/com/adyen/checkout/dropin/ui/action/ActionComponentDialogFragment.kt

```diff
@@ -51,6 +55,13 @@ internal class ActionComponentDialogFragment : DropInBottomSheetDialogFragment()
     private val actionConfiguration: GenericActionConfiguration by arguments(ACTION_CONFIGURATION)
     private lateinit var actionComponent: GenericActionComponent
 
+    private val requestPermissionLauncher =
+        registerForActivityResult(ActivityResultContracts.RequestPermission()) { isGranted: Boolean ->
+            if (!isGranted) {
```

Comment:
Something to improve UX wise: if this is granted we should retry the download, right now the shopper needs to click on the button again. Although I don't think it's an easy task so maybe add a TODO comment to handle this scenario later